mesos-docker/run/run.sh

```diff
@@ -522,6 +522,14 @@ function parse_args {
     fi
   fi
 
+  if [[ -n $SPARK_BINARY_PATH ]]; then
+   SPARK_FILE=${SPARK_BINARY_PATH##*/}
+  fi
+
+  if [[ -n $HADOOP_BINARY_PATH ]]; then
+   HADOOP_FILE=${HADOOP_BINARY_PATH##*/}
```

Comment:
for a specific problem there might be different solutions... but for this one i prefer to get stick to bash built-ins (a good principle) although code clarity may not be the best this is trivial i think... so i did some thinking there (LGTM).  http://stackoverflow.com/questions/22401091/bash-variable-substitution-vs-dirname-and-basename